src/main/java/com/synopsys/integration/detect/tool/binaryscanner/BlackDuckBinaryScannerTool.java

```diff
@@ -79,24 +81,21 @@ public BlackDuckBinaryScannerTool(final EventSystem eventSystem, final CodeLocat
         this.eventSystem = eventSystem;
     }
 
-    public boolean shouldRun() {
+    public Map<File, NameVersion> determineBinaryScanPaths(DockerTargetData dockerTargetData, NameVersion binaryScanOptionsNameVersion) throws DetectUserFriendlyException {
+        Map<File, NameVersion> binaryScanFiles = new HashMap<>();
+
+        NameVersion dockerTargetNameVersion = dockerTargetData.getNameVersion();
+        if (dockerTargetData.getContainerFilesystem().isPresent()) {
```

Comment:
Don't we want to scan both the squashed and the tar? Will only one be present at a time?

Reply:
Steve did some testing and we've determined that the best results are obtained by running bdba on the container filesystem tar, not the squashed image.  if the container filesystem file is not there, then we default to the tar provided to us by the user (if they did so). https://sig-confluence.internal.synopsys.com/display/integration/How+Docker+Tool+Interacts+with+Other+Tools

Reply:
Ok, so is it true we are only adding a single docker scan no matter how many of the DockerTarget fields are present?